src/components/.storybook/HotswapContainer/index.ts

```diff
@@ -0,0 +1,2 @@
+export { HotswapContainer } from './HotswapContainer';
+export { HotswapContainerProps } from './HotswapContainer';
```

Comment:
Is it the same `export { HotswapContainer, HotswapContainerProps } from './HotswapContainer';`

And after discussion with Eric, I recommend to bring all `type` s (i.e: `HotswapContainerProps `) in another file (i.e: `HotswapContainer.types.ts`) to avoid circular import issue.